e2e/benchmarks/model_config.js

```diff
@@ -89,16 +89,16 @@ function predictFunction(model, input) {
 }
 
 const benchmarks = {
-  'mobilenet_v2': {
+  'mobilenet_v3': {
```

Comment:
I don't know whether `mobilenet_v2` is still widely used. Do we need to keep `mobilenet_v2`?

Reply:
It's better to have both for a period of time so that we can have a thorough comparison before the transition.